charts/tekton/templates/tekton-triggers.yaml

```diff
@@ -1369,6 +1369,18 @@ spec:
         version: "v0.16.0"
     spec:
       serviceAccountName: tekton-triggers-controller
+      {{- with .Values.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
```

Comment:
Doesn't it make sense to set some default value, ex: `ops=true`?

Reply:
Sergey, Well that means that this solution (helm) should **know** how our infra is implemented and named - I'd like to avoid that.